FreeRTOS/iot-reference-esp32c3
# Update Security Feature guide to cover ESP-IDF latest version changes.

Update Security Feature guide to cover ESP-IDF latest version changes.
Description
-----------
This is to address this issue:  https://github.com/FreeRTOS/iot-reference-esp32c3/issues/60

There are some changes in menuconfig and command options in ESP-IDF 5.x.  The Secure Feature guide only applied to older version of the ESP-IDF.  

Test Steps
-----------
Tested the steps on a esp32-c3 board

Checklist:
----------


- [ x] I have tested my changes. No regression in existing tests.

Related Issue
-----------
[Metadata magic word does not match certificate errors when run on an encrypted device](https://github.com/FreeRTOS/iot-reference-esp32c3/issues/60)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
